alexa-news.php

```diff
@@ -67,17 +107,17 @@ private function endpoint_single_post( $id ) {
 		];
 	}
 
-	private function endpoint_content() {
-		$args = [
-			'post_type' => apply_filters( 'alexawp_post_types', [ 'post' ] ),
-			'posts_per_page' => 5,
+	private function endpoint_content( $args ) {
+		$news_posts = get_posts( array_merge( $args, [
+			'no_found_rows' => true,
 			'post_status' => 'publish',
-		];
-		$news_posts = get_posts( $args );
+		] ) );
+
 		$content = '';
 		$ids = [];
 		if ( ! empty( $news_posts ) && ! is_wp_error( $news_posts ) ) {
 			foreach ( $news_posts as $key => $news_post ) {
+				// Sounds a little strange when there's only one result.
```

Comment:
please prepend with `TODO` for easy future searching of known areas for improvement